src/virtualenv/create/via_global_ref/builtin/cpython/mac_os.py

```diff
@@ -70,7 +70,9 @@ def image_ref(cls, interpreter):
 class CPython2macOsFramework(CPythonmacOsFramework, CPython2PosixBase):
     @classmethod
     def can_create(cls, interpreter):
-        return not IS_MAC_ARM64 and super(CPython2macOsFramework, cls).can_describe(interpreter)
+        if not IS_MAC_ARM64 and super(CPython2macOsFramework, cls).can_describe(interpreter):
+            return super(CPython2macOsFramework, cls).can_create(interpreter)
```

Comment:
Why are we calling here twice the same method via super?

Reply:
Which method do you believe we're calling twice? AIUI CPython2macOsFramework.can_create() is resolving the method of the same name on a superclass, currently it resolves to ViaGlobalRefVirtualenvBuiltin.can_create()